### DISCUSSION TITLE
ondemand vs dom performance?

### DISCUSSION THREAD

**Discussion by Gun9niR** [simdjson/simdjson#2201] [Q&A] [answered]
Are there any numbers comparing the performance of ondemand vs dom frontend? Also I suppose simdjson DOM would still be much faster than rapidJSON right?

**Comment by lemire** [comment, marked as answer]
There is indeed, please see:

John Keiser, Daniel Lemire, On-Demand JSON: A Better Way to Parse Documents?
Software: Practice and Experience (to appear)
https://arxiv.org/abs/2312.17149
